sapi/fpm/tests/tester.inc

```diff
@@ -392,7 +392,7 @@ class Tester
         $configFile = $this->createConfig();
         $desc       = $this->outDesc ? [] : [1 => array('pipe', 'w'), 2 => array('redirect', 1)];
 
-        $cmd = [self::findExecutable(), '-F', '-y', $configFile];
+        $cmd = [self::findExecutable(), '-F', '-y', $configFile, '-dextension_dir=modules'];
```

Comment:
This is just specific to your setup where your extension dir is in modules so it cannot be applied in general. For example it produces warning in my local setup as I use some shared extensions that are not obviously in this directory.